### PR TITLE
Submit Coin98 Wallet

### PR DESCRIPTION
**Pull request by khiemsoft**

### CODE REVIEW

File: packages/wallet-nextjs/pages/_app.tsx [hunk 1]
```diff
@@ -35,7 +36,8 @@ function MyApp({ Component, pageProps }: AppProps) {
       new NightlyWalletAdapter(),
       new BitkeepWalletAdapter(),
       new TokenPocketWalletAdapter(),
-      new BloctoWalletAdapter({ network:WalletAdapterNetwork.Testnet })
+      new BloctoWalletAdapter({ network:WalletAdapterNetwork.Testnet }),
```
Comment (tom-hippo) [inline packages/wallet-nextjs/pages/_app.tsx:39]: I think it is better to align with other wallets and left out the default network by setting it to undefined

File: packages/aptos-wallet-adapter/src/WalletAdapters/Coin98Wallet.ts [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+  constructor({
+    // provider,
+    network = WalletAdapterNetwork.Mainnet,
+    timeout = 10000
+  }: Coin98WalletAdapterConfig = {}) {
+    super();
+
+    this._provider = typeof window !== 'undefined' ? window.coin98?.aptos : undefined;
+    this._network = network;
```
Comment (tom-hippo) [inline packages/aptos-wallet-adapter/src/WalletAdapters/Coin98Wallet.ts:100]: I think it is better to align with other wallets and left out the default network by setting it to undefined

File: packages/wallet-nextjs/pages/_app.tsx (hunk 1 above)
Comment (khiemsoft) [reply packages/wallet-nextjs/pages/_app.tsx:39]: This is from BloctoWalletAdapter. We do not change this. This is from the most recent main branch commit.

File: packages/aptos-wallet-adapter/src/WalletAdapters/Coin98Wallet.ts (hunk 2 above)
Comment (khiemsoft) [reply packages/aptos-wallet-adapter/src/WalletAdapters/Coin98Wallet.ts]: Commented fixed. Thanks for your review

**Pull request merged**